ci-operator/config/openshift/multiarch-manager-operator/openshift-multiarch-manager-operator-main.yaml

```diff
@@ -10,6 +14,34 @@ images:
 - dockerfile_path: Dockerfile
   from: ubi
   to: multiarch-manager-operator
+- dockerfile_literal: |
+    FROM src
+    COPY . /code
+    COPY operator-sdk /usr/local/bin/
+    RUN chmod -R g+rwX /code
+    WORKDIR /code
+  from: src
+  inputs:
+    operator-sdk:
+      as:
+      - operator-sdk
+      paths:
+      - destination_dir: .
+        source_path: /usr/local/bin/operator-sdk
+  to: operator-sdk-src
+operator:
+  bundles:
+  - as: multiarch-manager-operator-bundle
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: controller:latest
+    with: pipeline:multiarch-manager-operator
+promotion:
+  to:
+  - excluded_images:
+    - operator-sdk-src
+    namespace: origin
```

Comment:
@Prashanth684 one thing worth noting here. I added the promotion stanza towards the origin namespace and without targeting a 'release tag', which means no automatic branching will be executed for now. I can add it, but as we were still unsure about that, I wanted to postpone this a bit to have a discussion about it.

Reply:
ok..not a problem..we can discuss it.